addons/html_editor/static/src/main/list/list_plugin.js

```diff
@@ -695,8 +698,16 @@ export class ListPlugin extends Plugin {
             }
             element = element.parentElement;
         }
-        // Fully outdent LI.
-        this.liToBlocks(closestLIendContainer);
+        if (!closestLIendContainer.classList.contains("oe-nested")) {
+            // Remove LI marker on first backspace.
+            closestLIendContainer.classList.add("oe-nested");
+        } else {
+            // Fully outdent LI.
+            const ul = closestLIendContainer.parentNode;
+            const dir = ul.getAttribute("dir");
+            dir && closestLIendContainer.setAttribute("dir", dir);
```

Comment:
For nested list we can't be sure that dir attribute will be added to the parent UL/OL so I would suggest to use closestElement.
```suggestion
            const list = closestElement(closestLIendContainer, "ul[dir], ol[dir]");
            const dir = list.getAttribute("dir");
            dir && closestLIendContainer.setAttribute("dir", dir);
```

Also I was expecting a testcase for the fix in commit 2.